cmd/kyma/alpha/disable/module/module.go

```diff
@@ -0,0 +1,181 @@
+package module
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/kyma-project/cli/internal/clusterinfo"
+	"github.com/kyma-project/cli/internal/kube"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const defaultKymaName = "default-kyma"
+
+var kymaResource = schema.GroupVersionResource{
+	Group:    "operator.kyma-project.io",
+	Version:  "v1alpha1",
+	Resource: "kymas",
+}
+
+type command struct {
+	cli.Command
+	opts *Options
+}
+
+func NewCmd(o *Options) *cobra.Command {
+	c := command{
+		Command: cli.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "module [name] [flags]",
+		Short: "Disables a module in the cluster or in the given Kyma resource",
+		Long: `Use this command to disable Kyma modules active in the cluster.
+
+### Detailed description
+
+For more information on Kyma modules, see the 'create module' command.
+
+This command disables an active module in the cluster.
+`,
+
+		Example: `Examples:
+TODO
+`,
+		RunE:    func(cmd *cobra.Command, args []string) error { return c.Run(cmd.Context(), args) },
+		Aliases: []string{"mod", "mods", "modules"},
+	}
+
+	cmd.Flags().DurationVarP(
+		&o.Timeout, "timeout", "t", 1*time.Minute, "Maximum time for the operation to disable a module.",
+	)
+	cmd.Flags().StringVarP(
+		&o.Channel, "channel", "c", "",
+		"The channel of the module to use.",
+	)
+	cmd.Flags().StringVarP(
+		&o.Namespace, "namespace", "n", metav1.NamespaceDefault,
```

Comment:
kyma by default deployed in `kyma-system` namespace https://github.com/kyma-project/cli/blob/main/cmd/kyma/alpha/deploy/cmd.go#L75, I think we should align with it.

Reply:
I think by default is `kcp-system` at the moment? Should `list module` be adapted as well?

Reply:
yes, I think so, we should make all of them same by default in `kyma-system`

Reply:
I changed the default namespace to kyma-system in my last PR as this aligns it with the managed offering of kyma. `kyma-system` should be used for all kyma resources, but control-plane resources, e.g. lifecycle/module manager can still stay in kcp-system